common/test/data/scoreable/policies/course/policy.json

```diff
@@ -0,0 +1,64 @@
+{
+    "course/course": {
+        "advanced_modules": [
+            "lti", 
+            "lti_consumer", 
+            "library_content", 
+            "split_test", 
+            "conditional", 
+            "randomize", 
+            "drag-and-drop-v2", 
```

Comment:
Is there a reason drag-and-drop-v2 was not added to the test suite?  It is one of the block types that broke us earlier.